Apps/Admin/Client/Store/PatientDetails/PatientDetailsEffects.cs

```diff
@@ -47,7 +47,7 @@ public async Task HandleLoadAction(PatientDetailsActions.LoadAction action, IDis
 
             try
             {
-                PatientSupportDetails response = await this.SupportApi.GetPatientSupportDetailsAsync(action.QueryType, action.QueryString).ConfigureAwait(true);
+                PatientSupportDetails response = await this.SupportApi.GetPatientSupportDetailsAsync(action.QueryType, action.QueryString, action.RefreshVaccineDetails).ConfigureAwait(true);
```

Comment:
NIT:
`.ConfigureAwait` is no longer required in our code base. I feel its another item we can refactor out of code as we work with it.